components/Base/FileUpload.vue

```diff
@@ -1,23 +1,37 @@
 <template>
   <div>
     <label
-      class="flex h-20 w-full flex-col items-center justify-center rounded-3xl border-[2px] border-dashed bg-gray-100 p-3"
       :accept="accept"
-      draggable="true"
```

Comment:
I removed the `draggable="true"` attribute because it was creating a bug here. This item is not meant to be draggable. Draggable are the files to be dropped there. If you assign `draggable="true"` to an element, you can grab the element itself and move it around. We don't want to move our drop area anywhere.

Reply:
Previous behaviour with the attribute `draggable="true"`

https://user-images.githubusercontent.com/66744401/228256074-e15d723b-0143-4dc0-9be0-ee35c5308367.mov

Reply:
Current behaviour without the attribute:

https://user-images.githubusercontent.com/66744401/228256954-e06e9920-96cb-4f45-9a50-988bb867c141.mov